java/core/src/java/org/apache/orc/impl/ParserUtils.java

```diff
@@ -459,8 +466,8 @@ public static ColumnVector[] findColumnVectors(TypeDescription schema,
                                                  boolean isCaseSensitive,
                                                  VectorizedRowBatch batch) {
     List<String> names = ParserUtils.splitName(source);
-    ColumnFinder result = new ColumnFinder(schema, batch, names.size());
```

Comment:
It feels like it would be sufficient to add a new constructor to ColumnFinder that can also take an array of column vectors that in case the case of acid schema is the column vectors from the `row` column vector.